src/OpenLoco/GameCommands.cpp

```diff
@@ -164,6 +164,7 @@ namespace OpenLoco::GameCommands
         auto& gameCommand = _gameCommandDefinitions[esi];
         if ((flags & (GameCommandFlag::flag_4 | GameCommandFlag::flag_6)) == 0
```

Comment:
This is getting far too much logic for one if statement. Extract it to a function.

Reply:
Just extracting the expression to a static function? Sure. How should it be called? `commandRequiresUnpausingGame`?